src/activity.js

```diff
@@ -0,0 +1,128 @@
+class Activity {
+  constructor(activityData, id) {
+    this.data = activityData;
+    this.id = id;
+    this.user = this.findUserActivityData();
+  }
+
+  findCurrentDate() {
```

Comment:
Trying to figure out what this method does. It seems like we are not using it anywhere?